tests/plotting/test_plotting.py

```diff
@@ -123,6 +124,29 @@ def verify_cache_image(plotter):
                       f'{error}')
 
 
+@skip_not_vtk9
+@skip_no_plotting
+@pytest.mark.skipif(os.name == 'nt', reason="Windows CI testing segfaults on pbr")
```

Comment:
Does it work reliably outside CI? This sounds a bit scary :(

Reply:
Working on that.

Reply:
Works on windows.  I'm assuming that it's not working on windows on CI potentially with our Azure setup.  Off-screen works locally on windows too.